### PR TITLE
pythonPackages.trezor_agent: adding buildInputs required for GPG

### PR DESCRIPTION
**Pull request by mmahut**
###### Motivation for this change

Changes required for GPG support.

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

###### Notify maintainers

cc @np

### CODE REVIEW

File: pkgs/development/python-modules/trezor_agent/default.nix [hunk 1]
```diff
@@ -19,13 +21,13 @@ buildPythonPackage rec{
     sha256 = "e82bf000c1178b1a7612f2a90487eb34c6234d2edb15dc8e310ad875d8298690";
   };
 
-  propagatedBuildInputs = [ trezor libagent ecdsa ed25519 mnemonic keepkey semver ];
+  propagatedBuildInputs = [ trezor libagent ecdsa ed25519 mnemonic keepkey semver wheel pinentry ];
```
Comment (jonringer) [inline pkgs/development/python-modules/trezor_agent/default.nix]: generally wheel isn't needed after installation.
```suggestion
  nativeBuildInputs = [ wheel ];
  propagatedBuildInputs = [ trezor libagent ecdsa ed25519 mnemonic keepkey semver pinentry ];
```

Comment (mmahut) [reply pkgs/development/python-modules/trezor_agent/default.nix]: For some reason, this one fails at run time.

Comment (jonringer) [reply pkgs/development/python-modules/trezor_agent/default.nix]: interesting to need to use it at runtime.... but if that's the case, then disregard :)

Comment (mmahut) [reply pkgs/development/python-modules/trezor_agent/default.nix]: Indeed, pretty odd.

/cc @romanz as upstream.

**Comment by mmahut** [comment]
@GrahamcOfBorg build pythonPackages.trezor_agent python3Packages.trezor_agent

**Pull request merged**